### PR TITLE
N28: godot-rust

### PR DESCRIPTION
**Pull request by Bromeon**
_Part of #852_

### CODE REVIEW

**Comment by AngelOnFira** [comment]
I'll have to take another look at this one in a bit, it has two images so we'll likely have to find a way to make it one.

**Comment by AngelOnFira** [comment]
I think this might be a bit long. @Bromeon I'm going to make a commit with my suggested changes, then I'll get you to look it over :)

image

**Comment by AngelOnFira** [comment]
The paragraphs at the end are still long, but I'm inclined to let it slide for this section. @ozkriff @17cupsofcoffee what are your thoughts?

image

File: content/news/028/index.md [hunk 1]
```diff
@@ -74,6 +74,58 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+godot-rust ([GitHub][gd-github], [Discord][gd-discord], [Twitter][gd-twitter])
+is a Rust library that provides bindings for the Godot game engine.
+
+November has been a month of refactoring for godot-rust. The API was cleaned up
+across different locations, reducing confusion and making the library more
+accessible. The module simplification ([#811][gd-811]) continued initial efforts
+on the
+module structure, such as shorter paths and avoidance of redundant re-exports.
+The improvements are best expressed as a picture -- these are differences
+between version v0.9.3 and now:
```
Comment (17cupsofcoffee) [inline content/news/028/index.md:90]: This line doesn't make sense now that the picture is at the top - I think it'd read fine if we removed it?

**Review by 17cupsofcoffee — COMMENTED** [review]
The length is a bit much, but I'm happy to go ahead with this.

If we want to shorten it a bit more, we could drop the list of changed symbols? People can click through if they want to know what specifically has changed, so it's not essential to list here.

**Review by Bromeon — COMMENTED** [review]
Thanks for the review!

I shortened the text a bit:
* Removed symbol list, just mention 2 of 4 examples inline in the text itself
* Structured 3 refactoring changes as bullet list, making it a bit more compact
* Removed line:
   > Smaller changes include safety bugfixes or a `GodotString::format()` method

I rebased the changes onto latest `source` and fixed conflicts. Previous state would be bb4b86a3675e5ad2cfd85705447a0a40d8880399.

**Review by AngelOnFira — APPROVED** [review]
Thanks for the revision! Looks good to me now :)

**Comment by Bromeon** [comment]
Thank you! 🙂

**Pull request merged**